src/config/workload.rs

```diff
@@ -0,0 +1,412 @@
+use super::*;
+
+fn one() -> usize {
+    1
+}
+
+#[derive(Clone, Deserialize)]
+pub struct Workload {
+    #[serde(default)]
+    keyspace: Vec<Keyspace>,
+    #[serde(default)]
+    topics: Vec<Topics>,
+    threads: usize,
+    // zero is treated as unlimited
+    #[serde(default)]
+    ratelimit: u64,
+    /// if strict ratelimit is enabled, the test will stop if the rate is not
+    /// within 95% of the target
+    #[serde(default)]
+    strict_ratelimit: bool,
+    /// An optional p999 SLO in microseconds. Zero means no SLO enforcement.
+    #[serde(default)]
+    p999_slo: u64,
```

Comment:
where is this used?

Reply:
Removed the SLO and strict ratelimit functionality for the time being